tests/integration/github/test_issues.py

```diff
@@ -39,6 +39,21 @@ def test_create_issue(self):
         with self.assertRaises(NotImplementedError):
             project.create_issue(title=title, body=description, private=True)
 
+    def test_create_issue_with_assignee(self):
+        title = "This is an example issue"
+        description = "Issue description"
+        labels = ["label1", "label2"]
+        assignee = ["tunap"]
+        project = self.service.get_project(
+            namespace="BennettOSS", repo="Hacktoberfest19"
+        )
```

Comment:
Would be nice if we can use some really testing project -- e.g. packit-service/hello-world. (Ideally also for the previous test...;) (And since we are in that, what about splitting `private=True` test to a separate test case?)

Thanks!!